code-challenges.js

```diff
@@ -19,117 +19,148 @@ const fruit4 = "kiwi"
 
     // Expected output: 9
 
-    // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
-
-    // Pseudo code:
 
-    // const currentCohort = "Bravo 2023"
+    // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-const currentCohort = [Bravo, 2023]
+    // Pseudo code: 
+    // split Bravo 2023 to convert into a string ""
+    //Reverse "Bravo 2023" to read "3202 ovarB"
+    // join
 
-currentCohort.reverse(currentCohort)
+const currentCohort = "Bravo 2023" 
 
-    console.log(currentCohort)
+        // r = (currentCohort.split("")); (currentCohort.reverse(currentCohort)); (currentCohort).join
+        // c = (currentCohort.reverse(currentCohort))
+        // d = (currentCohort).join
+    
+        
+// console.log(r)
+    
 
     // Expected output: "3202 ovarB"
 
     // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
-    // Pseudo code:
+// Pseudo code:take in array use sort to sort temps largest to smallest
+    // input:  arrays
+    // output:  string
+    // preocess creat function to take arrays. consolelog my array(index) create let statement
+// output ----> .reverse is not a function all my notes said that I could do it the way I was doing it but it was not console logging. I kept getting reverse is not a function.
 
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
-    // const givenValueOne = 42
+    const givenValueOne = 42
+
+    for (let i = 0; i<myNumbers.length; i++) {
+    
+    console.log(givenValueOne[i])
+    }
+
+//----> undefined
+// undefined
+// undefined
+// undefined
+// undefined
+// undefined
+// undefined
+// undefined
+// undefined
+//  maybe i need one on one instructions to be able to break down these problems and get my process down so I know exactly what is going on with this code.
 
     // let givenValueOne = 42
     // let wanted = 2
-
-    // Let matchOne = myNumbers.find(item == 42) ;{
-    //     if (givenValue1 === item); return `${7}`
-    // }
-
-    // console.log(`last index number is 2`)
+    
 
     // Expected output: 7
 
-const givenValue2 = 10
+    const givenValue2 = 10
     // Expected output: 8
 
     // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
-    // Pseudo code:
+    // Pseudo code: take in array use sort to sort temps largest to smallest
+    // input: 2 arrays
+    // output: 1 new array
+    // preocess creat function to take in and sort arrays. using .sort to place all temps in order. use compare statements to compare temps against one another. 
 
```

Comment:
I think you have a good idea going here , just keep in mind that the input will be one array. You can use the advantage of a function in that you can apply that logic to any argument you provide but our input would be one array.